src/main/java/org/apache/maven/plugins/gpg/SignAndDeployFileMojo.java

```diff
@@ -149,7 +143,10 @@ public class SignAndDeployFileMojo extends AbstractGpgMojo {
 
     /**
      * The type of remote repository layout to deploy to. Try <i>legacy</i> for a Maven 1.x-style repository layout.
+     *
+     * @deprecated Maven3 does not support "legacy" (Maven1) layout anymore. This parameter is unused.
      */
+    @Deprecated
```

Comment:
I would like to remove such parameters.

With deprecated - we will have warning about it - the similar will be than we remove it and someone will use it - warning will be emitted

Reply:
Removed